github_stargazers/github_stargazers.py

```diff
@@ -7,53 +7,37 @@
 from github_stargazers.github import UsernameRepositoryError, TooManyRequestsHttpError, HTTPError
 
 
+class OutputPrintable(object):
+    @staticmethod
+    def print_stargazers(stargazers) -> None:
+        print("Stargazers:")
```

Comment:
This section will need to be updated a little after rebase according to my latest commits. For example for the case where `stargazers` is an empty list. Please see in `GitHub` what will be returned in that case and what will be printed in this case in `github_stargazers.py`. 
https://github.com/marius92mc/github-stargazers/pull/30/files#diff-d14ae9b4db48d38423876d733f778126L32